### PR TITLE
Add fields to Test model

### PR DESCRIPTION
**Pull request by rwongone**

### CODE REVIEW

**Comment by rwongone** [comment]
Will add File model and merge.

File: db/migrate/20170531005146_create_tests.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@ class CreateTests < ActiveRecord::Migration[5.1]
   def change
     create_table :tests do |t|
       t.string :name
+      t.string :filename
+      t.binary :file_contents
```
Comment (jchorl) [inline db/migrate/20170531005146_create_tests.rb]: this should really just point to a file, right?

Comment (rwongone) [inline db/migrate/20170531005146_create_tests.rb]: ah, yes

Comment (rwongone) [inline db/migrate/20170531005146_create_tests.rb]: Forgot to actually push all of the changes.

**Pull request merged**